homeassistant/components/risco/__init__.py

```diff
@@ -177,6 +184,22 @@ async def _update_listener(hass: HomeAssistant, entry: ConfigEntry) -> None:
     await hass.config_entries.async_reload(entry.entry_id)
 
 
+async def async_migrate_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> bool:
```

Comment:
We don't need a hard migration if we're only adding a new item in the options. We can add the option item during config entry setup.

Reply:
Removed